src/zarr/storage/remote.py

```diff
@@ -94,7 +94,11 @@ async def clear(self) -> None:
             pass
 
     async def empty(self) -> bool:
-        return not await self.fs._find(self.path, withdirs=True)
+        async for _path, _dirs, files in self.fs._walk(self.path):
+            # stop once a file is found
+            if files:
```

Comment:
Do included directories count as content? The original implied yes.

Reply:
We only check for files in the `LocalStore`:

https://github.com/zarr-developers/zarr-python/blob/5a134bf2c1c5e6655edd429ca554aa9699327463/src/zarr/storage/local.py#L101-L111